### PR TITLE
Fixed styles from Landing buttons

### PR DESCRIPTION
**Pull request by yehorovye**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ function Hero(): JSX.Element {
             return <div className="mt-3 rounded-md shadow sm:mt-0 sm:ml-3">
                 <Link href='/premium' passHref>
                     <a className="w-full flex items-center justify-center px-8 py-3 border border-transparent text-base
-                    font-medium rounded-md text-discord-blurple bg-white hover:bg-discal-light-grey md:py-4 md:text-lg
+                    font-medium rounded-md text-discord-blurple bg-opacoty-25 bg-white hover:bg-opacity-50 duration-200 md:py-4 md:text-lg
```
Comment (NovaFox161) [inline pages/index.tsx]: `opacoty` typo

File: pages/index.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ function Hero(): JSX.Element {
             return <div className="rounded-md shadow">
                 <Link href='/invite' passHref>
                     <a className="w-full flex items-center justify-center px-8 py-3 border border-transparent text-base
-                    font-medium rounded-md text-white bg-discord-blurple hover:bg-discal-green
+                    font-medium rounded-md duration-150 text-white bg-indigo-500 hover:bg-indigo-600
```
Comment (NovaFox161) [inline pages/index.tsx:35]: We want to keep the discord color codes rather than using indigo-x

**Review by NovaFox161 — CHANGES_REQUESTED** [review]
Thanks! There's some changes needed and please test with `yarn run`

File: pages/index.tsx (hunk 1 above)
Comment (yehorovye) [reply pages/index.tsx]: Ah yes srry

**Pull request closed without merging**